docs/advanced-topics/reconnecting-mid-game.md

```diff
@@ -0,0 +1,51 @@
+---
+id: reconnecting-mid-game
+title: Reconnecting Mid-Game
+---
+
+In a multiplayer game, clients may get disconnected from the server for a variety of reasons (network issues, application/device crashes, etc). For those reasons, you may want to allow your players to reconnect to the game.
+
+## Considerations
+
+Review the following considerations before allowing clients to reconnect in the middle of an active game:
+
+- Clients may have specific state and in-game data associated with them. If you want them to reconnect with that same state and data, implement a mechanism to maintain the association of state and data with the client for when they reconnect. See [Session Management](session-management.md) for more information.
+- When using [scene management](../basics/scene-management.md), remember that the synchronization process launches when connecting to a game. Therefore, all additive scenes currently loaded on the server will load without unloading anything on the client when the client's active main scene is the same as the server's. <!-- What is the negative affect of this? Like is it slower loading/initial lag, etc? Or is it just cleaner and better practice? -->
```

Comment:
This is really about one specific case where there are some effects that can be undesired in some cases:
If the active scene on the client is the same as the active scene on the server, the client will skip loading the server scene in single mode, but not additive scenes.
This means, that if the client was connected and over the course of the connection, some additive scenes were loaded by the server, the client could still have them loaded when disconnecting (if they don't switch scene when that happens). In that case, when reconnecting, if the server also has the same additive scenes currently loaded, the client would load every additive scene a second time.
But this wasn't very clear in my initial draft, so that's why @NoelStephensUnity suggested these changes : https://github.com/Unity-Technologies/com.unity.multiplayer.docs/pull/614#discussion_r876071897